numba_dpex/core/runtime/experimental/nrt_reserve_meminfo.cpp

```diff
@@ -0,0 +1,71 @@
+// SPDX-FileCopyrightText: 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "nrt_reserve_meminfo.h"
+
```

Comment:
Please add a file header using https://llvm.org/docs/CodingStandards.html#file-headers as the standard